--- include/freetype/config/ftoption.h.orig	2017-04-24 15:46:14.705889097 +0200
+++ include/freetype/config/ftoption.h	2017-04-24 15:46:12.077841302 +0200
@@ -122,7 +122,7 @@
   /* This is done to allow FreeType clients to run unmodified, forcing     */
   /* them to display normal gray-level anti-aliased glyphs.                */
   /*                                                                       */
-/* #define FT_CONFIG_OPTION_SUBPIXEL_RENDERING */
+#define FT_CONFIG_OPTION_SUBPIXEL_RENDERING
 
 
   /*************************************************************************/
